docs/use-cases/README.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Introduction
+---
+
+# Use Cases
+
+This section describes in more detail some of the use cases with current or planned support in the Benefits application.
+
+## Current work
+
+See our [Milestones][milestones] for current work tracked against specific features and use cases.
+
+## Product roadmap
+
+See our [Product Roadmap][roadmap] for more information on planned feature development and prioritization.
+
+<iframe src="https://docs.google.com/document/d/1IFoa8Ye0IXwGXXwxFjMrm1s3617Dbv6l-E-aCB0kgnA/edit#"
+        width="100%"
+        height="718"
+        frameborder="0"
+        marginheight="0"
+        marginwidth="0"
+        embedded=true>
+    Loading…
+</iframe>
+
+[milestones]: https://github.com/cal-itp/benefits/milestones
+[roadmap]: https://docs.google.com/document/d/1IFoa8Ye0IXwGXXwxFjMrm1s3617Dbv6l-E-aCB0kgnA/edit#
```

Comment:
@vykster Are you ok with us highlighting the roadmap?

Reply:
Important to note that only folks with access to the doc can click the link / see the embed.

Reply:
Good to go per [Vyki in Slack](https://cal-itp.slack.com/archives/C037Y3UE71P/p1658355046912929?thread_ts=1658344851.168449&cid=C037Y3UE71P)

Reply:
Merge away!